roles/test-utils/mining-device/Cargo.toml

```diff
@@ -1,11 +1,16 @@
 [package]
-name = "mining-device"
+name = "mining_device"
```

Comment:
Let's keep the `-` here, not an `_`.

Reply:
I should have written why I did that from the first place, sorry. 

Initially I did not change the hyphen, but then I got the following error: 
```
Caused by: library target names cannot contain hyphens
```
There is something about it here https://github.com/rust-lang/cargo/issues/11259 but I did not dig deep into it tbh.